services/director-v2/tests/conftest.py

```diff
@@ -161,7 +161,7 @@ def mock_env(
 
 
 @pytest.fixture(scope="function")
-def client(mock_env: EnvVarsDict) -> Iterable[TestClient]:
+async def client(mock_env: EnvVarsDict) -> Iterable[TestClient]:
```

Comment:
not sure I get the meaning of this change?

Reply:
Had some issues with the tests, tasks being created in different loops. This fixes it.

Reply:
it would be good to know why ?? :-)